lib/tests/formulae.rb

```diff
@@ -619,7 +619,7 @@ def formula!(formula_name, args:)
         if new_formula
           audit_args << "--new-formula"
         else
-          audit_args << "--git" << "--skip-style"
+          audit_args << "--git" << "--skip-style" << "--except=version"
```

Comment:
I reckon it's maybe worth leaving as-is here.

Reply:
Removed the change in the latest push 👍

Reply:
@MikeMcQuaid Following up on this now that I've seen this run on homebrew/core CI. Omitting this change is fine for `CI-syntax-only` PRs (which a PR that preemptively adds a redundant `version` should be) but CI will predictably fail on a normal run.

If someone were to submit a PR for a new `ammonite-repl` version between when Homebrew/homebrew-core#78878 is merged and Homebrew/brew#11491 is merged and tagged in a new version, CI would fail with the version audit error in that version bump PR.

If we also want to avoid the version audit error in this scenario, I'll need to create a follow-up PR to add this change.

Reply:
> If someone were to submit a PR for a new `ammonite-repl` version between when [Homebrew/homebrew-core#78878](https://github.com/Homebrew/homebrew-core/pull/78878) is merged and [Homebrew/brew#11491](https://github.com/Homebrew/brew/pull/11491) is merged and tagged in a new version, CI would fail with the version audit error in that version bump PR.

That seems reasonable/fine with me 👍🏻